Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -375,7 +375,7 @@ private class PhotonActionSheetHeaderView: UITableViewHeaderFooterView {
             self.siteImageView.backgroundColor = color
             self.siteImageView.image = self.siteImageView.image?.createScaled(PhotonActionSheetUX.IconSize)
         }
-        self.titleLabel.text = site.title.characters.count <= 1 ? site.url : site.title
+        self.titleLabel.text = site.title.isEmpty ? site.url : site.title
```

Comment:
This again looks like a change in logic. Its weird logic to begin with though so, I'm not sure if we should keep it.

Reply:
Disregard this comment. Let's keep your new logic.